# Experiment a299a034a03bdd3ec9f059f48123aa048edfcfce03ba0f02ef8819ab968ca1df
# Base 4a603c669103afff6e2ff8ff6ab2edc399b3cb42
diff --git a/src/Pnl_drop.cpp b/src/Pnl_drop.cpp
index 114bf5ffe..856e46334 100644
--- a/src/Pnl_drop.cpp
+++ b/src/Pnl_drop.cpp
@@ -634,10 +634,10 @@ long TDropDownButtonPanel::handle_mouse_move(long x, long y, int ctrl_key, int s
 
         if (this->drop_down_panel->scbar_panel->is_inside(x, y) != 0) {
             this->handle_mouse_up(1, x, y, ctrl_key, shift_key);
-            long list_width = this->drop_down_panel->list_panel->width();
-            long list_x = this->drop_down_panel->list_panel->xPosition();
+            long list_hit_x = this->drop_down_panel->list_panel->width() +
+                              this->drop_down_panel->list_panel->xPosition() - 1;
             return this->drop_down_panel->list_panel->handle_mouse_down(
-                1, list_width + list_x - 1, y, ctrl_key, shift_key);
+                1, list_hit_x, y, ctrl_key, shift_key);
         }
     }
 
